README.md

```diff
@@ -1,31 +1,73 @@
-# esp-openthread - WORK IN PROGRESS
+# openthread
 
-This is using the [Rust open source IEEE-802.15.4 driver](https://github.com/esp-rs/esp-ieee802154) and [OpenThread](https://openthread.io/) to provide a Thread implementation for ESP32 bare-metal Rust.
+[![CI](https://github.com/ivmarkov/esp-openthread/actions/workflows/ci.yml/badge.svg)](https://github.com/ivmarkov/esp-openthread/actions/workflows/ci.yml)
```

Comment:
we probably want to adjust the URLs later

Reply:
Yup. I just re-adjusted them, but likely one more step might be necessary post-merge.

I think we should also rename the `esp-openthread` repo to just `openthread` post-merge, and as previously discussed. I have reserved the `openthread` and `openthread-sys` names on `crates.io` in the meantime, and I'll move those to the `esp-rs` org.